app/views/pages/homepage/closing_homepage.html.erb

```diff
@@ -310,10 +310,5 @@
 </main>
 <script type="text/javascript">
     setupLanguageFilter();
-
     window.google = undefined;
```

Comment:
Do we need this line for anything else? Think we can pull this out.

Reply:
👍 Can be pulled, was for making turbolinks work w/ maps